src/components/Card.tsx

```diff
@@ -150,12 +182,17 @@ const cardFields: Fields<CardProps> = {
     type: "radio",
     options: [
       { label: "Left", value: "items-start" },
-      { label: "Center", value: "items-center"},
+      { label: "Center", value: "items-center" },
     ]
   }
 };
 
-export const Card = ({ image, heading, subheading, body, cta, alignment }: CardProps) => {
+export const Card = ({ image: imageField, heading, subheading, body, cta: ctaField, alignment }: CardProps) => {
+  const document = useDocument<LocationStream>();
+  // The null checks on the following lines are only necessary when upgrading a pre-existing field to use a mappable entity field
+  const image = resolveYextEntityField<ComplexImage>(document, imageField?.photo?.entityField)?.image
```

Comment:
Missing colons. Can you install the prettier extension in your vscode and run against your modified files?